detector/aws_route_specified_multiple_targets.go

```diff
@@ -7,20 +7,17 @@ import (
 
 type AwsRouteSpecifiedMultipleTargetsDetector struct {
 	*Detector
-	IssueType  string
-	TargetType string
-	Target     string
-	DeepCheck  bool
 }
 
 func (d *Detector) CreateAwsRouteSpecifiedMultipleTargetsDetector() *AwsRouteSpecifiedMultipleTargetsDetector {
-	return &AwsRouteSpecifiedMultipleTargetsDetector{
-		Detector:   d,
-		IssueType:  issue.ERROR,
-		TargetType: "resource",
-		Target:     "aws_route",
-		DeepCheck:  false,
-	}
+	nd := &AwsRouteSpecifiedMultipleTargetsDetector{Detector: d}
+	nd.Name = "aws_route_specified_multiple_targets"
+	nd.IssueType = issue.ERROR
+	nd.TargetType = "resource"
+	nd.Target = "aws_route"
```

Comment:
__[gometalinter]__
* [goconst] 8 other occurrence(s) of "aws_route" found in: aws_route_invalid_nat_gateway.go:23:14 aws_route_invalid_network_interface.go:23:14 aws_route_not_specified_target.go:17:14 aws_route_invalid_instance.go:23:14 aws_route_invalid_egress_only_gateway.go:23:14 aws_route_invalid_route_table.go:23:14 aws_route_invalid_gateway.go:23:14 aws_route_invalid_vpc_peering_connection.go:23:14

<!-- Comment by [SideCI](https://sideci.com) -->